src/SFA.DAS.AANHub.Application.UnitTests/StagedApprentices/Queries/GetStagedApprenticeQueryHandlerTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.StagedApprentices.Queries;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.StagedApprentices.Queries
+{
+    [TestFixture]
+    public class GetStagedApprenticeQueryHandlerTests
+    {
+        [Test]
+        public async Task Handle_GetStagedApprenticeMember()
```

Comment:
Add a test to return a ValidatedResponse with null result.

Reply:
Added Test